_rules/object-has-acessible-name-8fc3b6.md

```diff
@@ -38,11 +41,16 @@ The `object` element is not rendered for presentational purposes. If the `object
 
 ## Accessibility Support
 
-Non-supported media formats make screen readers render the text content of the element instead of other attributes.
+Non-supported media formats make screen readers render the text content of the element instead of other attributes. `object` elements without an accessible name are ignored by assistive technologies unless they have an [explicit role][].
 
 ## Background
 
-Testing that the [accessible name][] describes the purpose of the `object` element is not part of this rule and must be tested separately.
+This rule is testing that the following 'object' attributes are followed:
+
+- The purpose is in the [accessible name][] but is not part of this rule and must be tested separately; and 
+- When the object resource is not loaded the fallback content is rendered; and
+- Valid `data` defines the [accessible name][] that is announced by the screen reader; and
+- The [accessible name][] is defined depending on the role of the `data` attribute.
```

Comment:
I'm also not sure that the (semantic) role of the `object` element actually has an impact on how its accessible name is computed.

Reply:
If the object is an image, then the accessible name can be computed from an alt, but a MIME might use an aria-label. I could add that in as an example to clarify?